src/client/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -201,20 +197,6 @@ public PublicClientApplicationBuilder WithWindowsBrokerOptions(WindowsBrokerOpti
             return this;
         }
 
-#if WINDOWS_APP
-        /// <summary>
-        /// Flag to enable authentication with the user currently signed-in on Windows.
-        /// </summary>
-        /// <param name="useCorporateNetwork">When set to true, the application will try to connect to the corporate network using Windows Integrated Authentication.</param>
-        /// <returns>A <see cref="PublicClientApplicationBuilder"/> from which to set more
-        /// parameters, and to create a public client application instance</returns>
-        public PublicClientApplicationBuilder WithUseCorporateNetwork(bool useCorporateNetwork)
-        {
-            Config.UseCorporateNetwork = useCorporateNetwork;

```

Comment:
Remove this variable from here: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/src/client/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs#L64?